int32 group_index
string uri
float32 Yaw_est